engine/src/main/java/org/terasology/engine/core/PathManager.java

```diff
@@ -140,6 +140,12 @@ public static PathManager getInstance() {
         return instance;
     }
 
+    static PathManager setInstance(PathManager pathManager) {
```

Comment:
This weakens the property of it being a singleton :thinking: Sure, this is not publicly visible, but I'd at least would like to see a JavaDoc comment to emphasize that this is for testing-only.

I guess the "right thing" to do here would be to not make it a singleton at all but rather have a "dummy factory" that produces exactly on instance in normal operation, but can be used to produce stubs when testing...